static/images/integrations/logos/google-hangouts.svg

```diff
@@ -1 +1 @@
jUxDJYzhqfQxxum8HKUsQBWIocIhfuFUa8jW29W+zwJXGrdI+1gZBDI7iZrnLdpBWBtEohs7ax3BnzdstcKWdOcTO26G9u9G9/heE3F+8xokjhq+v3H07PBjdvLdMDAGIwduct90whs62iOHJuY5/60WusrOH2Xk77Cw2iCIxnNU5hp23gxQZRKEYeG74+so7xPBGzAiIIUiduab1ZoS+H7sWCkEqd7ihttgOB8633xiKXV08RJuvvEMMB/mMghiCEcNlB0d4fRzEEKzWfFM6McLxZ4hiIUgVDzn2gLfZDieMjaTelTV0yOu3Q5xtYjh53s+PhRgm4blhheO3TAVDkIhhawNjPPa9ghgm4rnBxeAY349noxjWI4bVnt8y1byiLh322u1ADG4ujPLxmIhhMmJwc3GUv2+IoiFIlWNAJv9/7yCGRdgObhxG+XNcxLAIMbhxGuVt1xjWI4aMyo7tagw8RDfiOM6SoyOGF1+95Bm9wHmc5cZHDIg09zf3XeDxvNA9BrbDdSVGSAwHXqHEmXQUNNL0Y6z9SZJEDLWkHqVXDOu2AzGECBxr2nESw4FXSHv2AgWPNd1DtectEjE0M+H2OdVYiQGrpQmCGE68SpqzNtGk7zMpRksMJ14lxRmbbOLSXfoM4f2RKjHgqmUjrv/zhW/EEGrydbJkzBExrNkOxBBqwTfNqbdMUVuBGBpadAcxbdzEMPAqu8aw0JSRE8PAq+wcw8Lny9BbpsgHZ2JoavGHLWGjJ4bBVyGGpdw3RPTHqcTQVIIYJOcNQQwXXmXnGBJxOw3EcOFVdo8hyXaQHG6ZZvzEmRgaSxSDdHFD9PnrF/eIYYqE18/QhpgZwtztMO+NbR9DUqdPCzE4vBIxpNwO0okNMfv2iBgaSxqDdCCIFc8JxNBY4hikN7dMxOD4SsSQPgbpyYZY9ekRMTRWIAbpQRD9Y5j7Bomhll9BrPyZAjE0V2Q7SNLn6h+uEUNzhWKQFv8aGmJorlgM0sIgiKG5gjFIi34v05wY5p8RYvhSNAZpwYYgBmQ29TQSwwYKbwdp4i1TfAyEsFzxGKRJG4IYNtAgBmlCEMSwgSYxSMG3TLExrDkLxHCnUQxS4IYghg00i0EKCoIYNtAwBinglikuBkJIo2kMkvOGIIYNNI5BcgyCGNCByy1TTAzrvhURwxPNt4PkEESnGAjhhQ1ikC4G4R8DWyGlTWKQBoPotBUkYnhpoxikgSA6bQWJGF7aLAbpZBCdYiCENzaMQToRhG8MbIXUNo1BOvhzCGLYyMYxSAc2hF8MhJDe5jFIb4LwiWH9lInhgPWnKYWnlwoxbGT9aUrj4Ya4HsP6CRPCQetPVSp/XDbEsJH1pyqdXxuiQwgSMRyW43Sl8yldjSHHZAnhhBynLKXP8RjyTJUYTshz2lIa/NuuOaZKCCflOG2pnQwiz0SJ4aQ8py61g0HkmSYhDMhz+tJ7E0SuSRLDgFynML0nQeSaIiEMynUaS7gLIt8EiWFAvtNYhgki1xQJYVCu01jOZ8YJEsOgfKeynCX/xtwzhHABMbhIEQQhXEQMbpZfisRwETG4WrYhCMEBMbhbclkSgwNiCDF1QxCCA0IINSUIQnBCDOFCgyAER8QwRdglSwyOiGEa9w1BCM6IYSq3IAghADFMdzkIQghACMsMB0EIQYhhqaHLmhiCEMNypzYEIQQihhQOBUEIgQghlZdBEEIwYkjnYRCEMAExpPRbEIQwCTGk9SkRwjSEkN6NGCYhhhLIYQZiKCPFLxloixDKYUNEIYaSCCICMZTFLZMnQiiPDeGFGFpgQ1xFCK2wIa4ghnYIYhQxtMQt01mE0Bob4gxiaI8NcQQhbIMN8Q4xbIUN8QwhbIkN8QgxbIsNYRHC9tgQ34gBYkMQAn6z94YgBtzZc0MQAp7Yb0MQA17YZ0MQAg7oHwQh4ITet0zEgJN6bghCwKBeQRACLupzy0QMcFB/QxACHNUNghAQoF4QhIBAtZ4hiAHBamwIQsAkuYMgBEyWMwhCwCK5giAELJYjCEJAEmuDIAQksyYIQkBSc4MgBCQ3JwhCQBGxQRACiokJghBQlG8QhIDifIIgBDRxLQhCQDNjQRACmjoXBCGgufdBEAE28jwIQsCG/gyCELCxn0EQASBJuhED8L9av3UDCEYQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgEEQgPEfpYY4ibquNwMAAAAASUVORK5CYII=" transform="matrix(942.2 0 0 -1383.2 2235.9 1783.1)" preserveAspectRatio="none" height="1" width="1"/></g><g clip-path="url(#u)" transform="matrix(.07014 0 0 -.07014 16 256)"><image xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAU8AAADhCAYAAABIrDfYAAAABHNCSVQICAgIfAhkiAAABH9JREFUeJzt2kFy2zAUREHF5fvf1mtnkVQcJ5JNDQECH+g+ARasVwOSP97e3t5vMKn3d48nc3oZfQB4RDiZmXgCBMSTKVmdzE48mY5wUoF4AgTEk6lYnVQhngAB8WQaVieViCdTEE6qEU+AgHgynNVJReLJUMJJVeIJEBBPhrE6qUw8AQLiyRBWJ9WJJ5cTTlYgngAB8eRSVierEE8uI5ysRDy5hHCyGv
```

Comment:
Why did this get changed in your PR?

Reply:
`svgo` was used to optimize the svgs in `static/images/integrations/logos/`. Other than `honeybadger.svg` they were also optimized so I made a different commit for them.